CMakeLists.txt

```diff
@@ -73,10 +73,14 @@ add_executable(astra_camera_node
 target_link_libraries(astra_camera_node astra_driver_lib)
 add_dependencies(astra_camera_node astra_openni2)
 
-install(TARGETS astra_camera_node astra_wrapper astra_driver_lib
+install(TARGETS astra_camera_node
   DESTINATION ${CMAKE_INSTALL_PREFIX}/bin/
 )
 
+install(TARGETS astra_wrapper astra_driver_lib
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/
```

Comment:
I agree, it's worth a retroactive change to make the code look like what @dirk-thomas linked.